src/test/java/stepDefinition/ShoppingCartPageStepDef.java

```diff
@@ -0,0 +1,64 @@
+package stepDefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.junit.Assert;
+import pages.ShoppingCartPage;
+
+import static helpers.ElementSearchUtils.getElementByName;
+import static helpers.ElementSearchUtils.getPage;
+import static org.hamcrest.core.Is.is;
+import static utils.TestUtils.getPageObjectName;
+
+public class ShoppingCartPageStepDef extends AbstractStepDef {
+
+    ShoppingCartPage shoppingCartPage = new ShoppingCartPage(webDriver);
+    String inputQuantity;
+
+    @Then("^'(.*)' product is displayed$")
+    public void productIsDisplayed(String product) {
+        waitForPageLoaded();
+        Assert.assertTrue("The product is not found", getElementByName("productName").isDisplayed());
+    }
+
+    @Then("^'(.*)' product quantity changed correct$")
+    public void checkProductQty(String productName) {
+        Assert.assertEquals("The product quantity doesn't matched", inputQuantity, getElementByName("quantityField").getAttribute("value"));
```

Comment:
Please don't use a such long lines it's not readable at all

Reply:
Done.
Thank You!